bindings/core/src/method/wallet.rs

```diff
@@ -438,6 +438,13 @@ pub enum WalletMethod {
         #[serde(default)]
         options: Option<TransactionOptions>,
     },
+    /// Send mana.
+    /// Expected response: [`SentTransaction`](crate::Response::SentTransaction)
+    SendMana {
+        params: Box<SendManaParams>,
```

Comment:
Clippy asked for it?

Reply:
Not really ... I just assumed it would complain, and we do it already for some other variant. Also ... even if it's not complaining atm, it could in the future, should we ever add more params.

Reply:
If it actually does not complain, I would remove it since it has a runtime cost. And this is not breaking change so we can always change later if it starts complaining at some point.

Reply:
It didn't complain so I removed it.